docs/targets/awslambda.md

```diff
@@ -3,11 +3,11 @@
 This event Target receives [CloudEvents][ce] over HTTP and invokes an AWS
 Lambda function.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. AWS API key and secret
+- AWS API key and secret
```

Comment:
These can be turned to *

Reply:
If we agree that we will use `-` in the future and address lists which are already unordered in a separate PR, I think those changes can stay.

I'm not feeling strongly about it, just want to avoid discouraging people for something we will replace anyway 🙂 

If we decide not to use `-`, then it's a different story of course.

Reply:
@antoineco Thomas is creating a "rules" document that will have these rules lined out. It will say to use `*`. Hoping the addition to the rules page will help standardize these more. :)

Reply:
The consensus is to use -